vendor/github.com/stoewer/go-strcase/camel.go

```diff
@@ -27,6 +27,9 @@ func camelCase(s string, upper bool) string {
 				buffer = append(buffer, toUpper(curr))
 			} else if isLower(prev) {
 				buffer = append(buffer, curr)
+			} else if isUpper(prev) && isUpper(curr) && isLower(next) {
+				// Assume a case like "R" for "XRequestId"
+				buffer = append(buffer, curr)
```

Comment:
looks like this is only used by CEL... not sure the impact of this change, if any

https://github.com/kubernetes/kubernetes/blob/429edc5f25efb76743677217326dc19360ab5c2f/vendor/github.com/google/cel-go/common/types/map.go#L206-L207

cc @jpbetz for input on whether this is ok

Reply:
We do not use this in Kubernetes.

We [use our own types](https://github.com/kubernetes/kubernetes/tree/master/staging/src/k8s.io/apiserver/pkg/cel/common) for CRDs, and we use `stringIfaceMapAccessor` in DRA.

cc @TristonianJones for potential impact to other CEL users.

Reply:
thanks for confirming @jpbetz !

Reply:
@dims This change shouldn't impact Kubernetes. The code in question is used to try to map from a CEL Map value to a Golang struct during calls to `ConvertToNative` which I don't think are in the K8s call path, or at least won't be used for those purposes. This should be safe to update.

Reply:
@TristonianJones thanks for confirming!